bin/cli.js

```diff
@@ -40,12 +40,15 @@ const argv = require('yargs')
         async (argv) => {
             await testRunner({
                 sourceDir: argv.s,
-                reporter: argv.reporter
+                reporter: argv.reporter,
+                globalSetupFile: argv.g,
             });
         }
     )
     .describe('s', 'Path to brs files (if different from source/)')
     .alias("s", "source")
+    .describe("g", "Path to a global setup file")
```

Comment:
I'm torn on whether to call this `-r`/`--require` to align with [mocha's `--require`](https://mochajs.org/#-require-module-r-module), if only because we aren't looking for a setup/teardown function to execute in this file.  But considering that BrightScript doesn't really have a concept of `require` anyway, that also feels kind of weird.

🤔 I'm not sure what the right name is — any thoughts?

Reply:
I think that `-r`/`--require` would be fine:
1. If you disassociate the term `require` from how it behaves in Javascript, then it still kind of makes sense -- we are "requiring" that this file be run before the tests.
2. (I know this shouldn't be an argument, but it still kinda is) I would wager that a lot of BrightScript developers are also Javascript developers and thus familiar with `require`.
3. Even if someone doesn't understand what `require` means, and is not a Javascript developer, the usage will be in the docs for them to look at 😉